source/optional_features/corporate_card_surcharges/index.html.md.erb

```diff
@@ -0,0 +1,57 @@
+---
+title: Corporate card surcharges 
+weight: 134
+---
+
+# Corporate card surcharges  
+
+GOV.UK Pay supports adding a surcharge when a user pays with a corporate
+credit or debit card. You can [contact GOV.UK Pay
+support](/support_contact_and_more_information/#contact-us) to ask the team to
+set this up for you. 
+
+Each surcharge is a flat amount added to a payment, not a percentage. 
+
+You can set a different corporate surcharge for each of the following 4
+corporate card types: 
+
+* non-prepaid credit cards 
+* non-prepaid debit cards
+* prepaid credit cards
+* prepaid debit cards 
+
+When a user enters a card number in their payment journey, the GOV.UK Pay
+platform checks to determine if it's one of the 4 corporate card types. If a
+surcharge is applicable, GOV.UK Pay will inform the user that a corporate
+surcharge of a certain amount will be added to the total. If the platform
+cannot determine the exact type of a card being used, it will not apply a
+surcharge.
+
+You can sign in to the [GOV.UK Pay admin
+tool](https://selfservice.payments.service.gov.uk/login) (link opens in new
+window) to see the payment amount and the surcharge amount listed separately. 
+
+If a payment has a surcharge applied, the full amount available for refund
+includes the surcharge. You can also use the [partial refund
+feature](/refunding_payments/#payment-refund-status-and-partial-refunds) to
+only refund the total amount excluding the surcharge. 
+
+If a corporate surcharge is applied to a particular payment, you can see this
```

Comment:
Similar to the comment above, we probably want this to be “corporate card surcharge”